### ISSUE TITLE
About training implementation details

### ISSUE THREAD

**Issue by lxn96** [lixiang-ucas/FSODM#7]
Hello, I would like to ask how many epochs or iters to train in the meta-training stage and the meta-fine-tuning stage, and what is the batch size and learning rate. Thanks!
